src/Features/LanguageServer/Protocol/Features/Diagnostics/DiagnosticAnalyzerService.cs

```diff
@@ -77,11 +77,6 @@ public static bool IsGlobalOptionAffectingDiagnostics(IOption2 option)
 
         public void Reanalyze(Workspace workspace, IEnumerable<ProjectId>? projectIds, IEnumerable<DocumentId>? documentIds, bool highPriority)
```

Comment:
i intend to remove this method in followup.  it was a bit viral though, so i kept it for now.